src/components/Filters/ToggleFilter.tsx

```diff
@@ -3,36 +3,45 @@ import { Filter } from "src/components/Filters/types";
 import { Switch } from "src/inputs/Switch";
 import { TestIds } from "src/utils/useTestIds";
 
-export type ToggleFilterProps = { label?: string; enabledValue?: boolean; defaultValue?: boolean };
+export type ToggleFilterProps<V> = {
+  label?: string;
+  onValue?: V | undefined;
+  offValue?: V | undefined;
+  defaultValue?: V | undefined;
+};
 
-export function toggleFilter(props: ToggleFilterProps): (key: string) => Filter<boolean> {
-  return (key) => new ToggleFilter(key, props);
+/**
+ * Provides a two-state "on/off" filter.
+ *
+ * By default the on/off values are `on === true` and `off === undefined`.
+ *
+ * You can flip the on/off values by passing `onValue: false`, in which case
+ * `on === false` and off === undefined`.
+ *
+ * Or you can set on/off directly, by passing both `onValue` and `offValue`.
+ *
+ * @param props
```

Comment:
Please remove this, probably added by WebStorm.

Reply:
Done!